test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -805,7 +803,11 @@ class V2DeparturesControllerSpec
           messageId1 <- arbitrary[MessageId]
           messageId2 <- arbitrary[MessageId]
           messageId3 <- arbitrary[MessageId]
-        } yield Seq(generateResponse(messageId1), generateResponse(messageId2), generateResponse(messageId3))).sample.value
+        } yield Seq(
+          genMessageSummary(Some(messageId1)).arbitrary.sample.value,
+          genMessageSummary(Some(messageId2)).arbitrary.sample.value,
+          genMessageSummary(Some(messageId3)).arbitrary.sample.value
+        )).sample.value
```

Comment:
I think you missed the point with the use of arbitrary it was to replace the use of the messageId1, messageId2, messageId3 calls not just a relocation of the method you'd written

Reply:
I think I did. Can you provide me with the code you mean?

Reply:
see an example in the comment for CommonGenerators